packages/moonstone/GridListImageItem/GridListImageItem.js

```diff
@@ -132,6 +140,7 @@ const GridListImageItemBase = kind({
 				{...rest}
 				captionComponent={captionComponent}
 				css={css}
+				data-webos-voice-intent={voiceIntent || 'Select'}
```

Comment:
What is the reason to apply the default value here rather than via `defaultProps`?

Reply:
It seems that using defaultProps is better way and I will change it. Thanks~